install/install.sh

```diff
@@ -121,7 +121,7 @@ read -e -r -p "GitHub Owner Name : " -i "CCExtractor" github_owner_name
 read -e -r -p "GitHub repository : " -i "ccextractor" github_repository
 read -e -r -p "Email Domain : " -i "${config_server_name}" email_domain
 read -e -r -p "Email API key (Generate one here https://www.mailgun.com/) : " -i "" email_api_key
-hmac_key=$(< /dev/urandom tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
+hmac_key=$(head -80 /dev/urandom | LC_ALL=c tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
```

Comment:
Please work with branches so changes from your previous PR's aren't in here. This will prevent us merging this one if others wouldn't be merged.

Reply:
Ok, Sir, I will remove previous commits from this PR to ensure this.